src/entities/Place/model.ts

```diff
@@ -548,10 +548,14 @@ export default class PlaceModel extends Model<PlaceAttributes> {
       )}
 
       WHERE
-        p.disabled is false AND world is true
+        p.world is true
+        ${conditional(!!options.disabled, SQL`AND p.disabled is true`)}
+        ${conditional(!options.disabled, SQL`AND p.disabled is false`)}
         ${conditional(
           options.names.length > 0,
-          SQL`AND world_name IN ${values(options.names)}`
+          SQL`AND LOWER(world_name) = ANY(ARRAY[${values(
```

Comment:
If I'm not wrong, when giving an array to the SQL function, it is converted as a PostgreSQL array:
```suggestion
            AND LOWER(world_name) = ANY($1)
```
Did you try it by any chance?

Reply:
I'm also not seeing an index over the lower version of `world_name`, if we use lower here, if I'm not wrong, we might be ignoring the index. Ignoring the index might have a big impact on the query.
It might be possible to apply a lower to all the tables via an update in a migration or modify the current index to be lower cased.